services/common/src/constants/index.ts

```diff
@@ -17,3 +17,4 @@ export * from "./environment";
 export * from "./fileTypes";
 export * from "./utils";
 export * from "./enums";
+export * from "./forms";
```

Comment:
exporting this as an enum does deal with the eslint import error mentioned above (expand to view), and is identical in usage to what we were doing.